src/test.cc

```diff
@@ -75,10 +75,12 @@ char* mkdtemp(char* name_template) {
 
 string GetSystemTempDir() {
 #ifdef _WIN32
-  char buf[1024];
```

Comment:
The change to the buffer size seems like it could be done immediately, without anyone bike shedding.

Why not make a PR that only changes the 1024 -> MAX_PATH, so that this fix can get included immediately?

Reply:
Also, I'm not certain this is an improvement.

https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#maximum-path-length-limitation

MAX_PATH is 260 ...?

Reply:
@jlonnberg